src/gui/transferlistfilterswidget.cpp

```diff
@@ -393,7 +395,14 @@ void TrackerFiltersList::downloadFavicon(const QString& url)
 
 void TrackerFiltersList::handleFavicoDownload(const QString& url, const QString& filePath)
 {
-    QString host = getHost(url);
+    QString host;
+
+    if (url.startsWith(GOOGLE_FAVICON_URL)) {
+        host = url.mid(url.indexOf("domain=") + 7);
```

Comment:
You can do url.mid(url.indexOf(GOOGLE_FAVICON_URL) + GOOGLE_FAVICON_URL.size());